csrc/custom_all_reduce.cuh

```diff
@@ -22,24 +27,38 @@
     }                                                               \
   } while (0)
 
-namespace vllm {
-
+// Maximal number of blocks in allreduce kernel.
 constexpr int kMaxBlocks = 36;
+
+// Default number of blocks in allreduce kernel.
+#ifndef USE_ROCM
+const int defaultBlockLimit = 36;
+CUpointer_attribute rangeStartAddrAttr =
+    CUDA_POINTER_ATTRIBUTE_RANGE_START_ADDR;
```

Comment:
Looking at the build logs, I think this needs to be `CU_POINTER_ATTRIBUTE_RANGE_START_ADDR`